articles/extensions/deploy-cli/guides/test-locally.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Test Deploy CLI Operations Locally
```

Comment:
to be honest, I'm not sure that this actually belongs in the docs. it's fine on the repo readme, but seems out of place here.

Reply:
I removed this file and all the links to it.